apps/ehr/src/features/css-module/pages/ProceduresNew.tsx

```diff
@@ -0,0 +1,569 @@
+    stateMutator: (value: string, state: State) => void,
+    error = false
+  ): ReactElement => {
+    return (
+      <FormControl fullWidth sx={{ backgroundColor: 'white' }} size="small" error={error}>
+        <InputLabel id={label}>{label}</InputLabel>
+        <Select
+          label={label}
+          labelId={label}
+          variant="outlined"
+          defaultValue=""
+          onChange={(e) => updateState((state) => stateMutator(e.target.value, state))}
+        >
+          {options.map((option) => {
+            return (
+              <MenuItem key={option} value={option}>
+                <Typography color="textPrimary" sx={{ fontSize: '16px' }}>
+                  {option}
+                </Typography>
+              </MenuItem>
+            );
+          })}
+        </Select>
+        {error ? <FormHelperText>{REQUIRED_FIELD_ERROR_MESSAGE}</FormHelperText> : undefined}
+      </FormControl>
+    );
+  };
+
+  const otherTextInput = (
+    parentLabel: string,
+    parentValue: string | undefined,
+    stateMutator: (value: string, state: State) => void
+  ): ReactElement => {
+    if (parentValue !== 'Other') {
+      return <></>;
+    }
+    return (
+      <TextField
+        label={'Other ' + parentLabel.toLocaleLowerCase()}
+        size="small"
+        onChange={(e: any) => updateState((state) => stateMutator(e.target.value, state))}
+      />
+    );
+  };
+
+  const radio = (
+    label: string,
+    options: string[],
+    stateMutator: (value: string, state: State) => void,
+    error = false
+  ): ReactElement => {
+    return (
+      <FormControl error={error}>
+        <FormLabel id={label}>{label}</FormLabel>
+        <RadioGroup
+          row
+          aria-labelledby={label}
+          onChange={(e) => updateState((state) => stateMutator(e.target.value, state))}
+        >
+          {options.map((option) => {
+            return <FormControlLabel key={option} value={option} control={<Radio />} label={option} />;
+          })}
+        </RadioGroup>
+        {error ? <FormHelperText>{REQUIRED_FIELD_ERROR_MESSAGE}</FormHelperText> : undefined}
+      </FormControl>
+    );
+  };
+
+  return (
+    <Stack spacing={1}>
+      <PageTitle label="Document Procedure" showIntakeNotesButton={false} />
+      <InfoAlert
+        text="Please include body part including laterality, type and quantity anesthesia used, specific materials (type
+              and quantity) used, technique, findings, complications, specimen sent, and after-procedure status."
+      />
+      <AccordionCard>
+        <Stack spacing={2} style={{ padding: '24px' }}>
+          <Box style={{ display: 'flex', alignItems: 'center' }}>
+            <Checkbox
+              onChange={(_e: any, checked: boolean) => updateState((state) => (state.consentObtained = checked))}
+            />
+            <Typography>I have obtained the Consent for Procedure *</Typography>
+          </Box>
+          {dropdown(
+            'Procedure type *',
```

Comment:
similarly here for the *, could add another optional property for required in the `dropdown` function